authors/wubwub/05-SegWit2x/hr_HR/final.md

```diff
@@ -0,0 +1,181 @@
+
+Kao što već vjerojatno znate, svaka strana je dobila što želi, BItmain je forkao Bitcoin na Bitcoin Cash, a Core developeri su uz konsenzus mreže implementirali SegWit u obliku soft forka (što znači da nije potrebna nikakva intervencija krajnjeg korisnika).
+
+## Segwit2x kao korporativni napad na Bitcoin?
+
+Uz ove gore navedene probleme s SegWit2x nadogradnjom, također postoji problem ne postojanja konsenzusa oko SegWit2x. Naime, sve te kompanije i rudari su potpisali NYA dogovor, no nitko nije pitao Core developere slažu li se s time, no ni Core developeri nisu toliko bitni kao što su bitni sami korisnici Bitcoina.
+
+Ljepota Bitcoina kao decentraliziranog protokola je baš u tome što je demokratski, svatko može glasati tako da kod kuće ima Bitcoin čvor koji primjenjuje stroga pravila po kojima neki blok smatra validnim. Tako da ako rudari aktiviraju SegWit2x i rudare veće blokove, čvorovi ih neće prihvatiti jer po pravilima Bitcoina ti blokovi nisu validni.
+
+Rudari će se automatski forkati na drugi lanac, dok će rudari koji rudare po pravilima ostati na pravom lancu.
+
+Situacija se trenutno poboljšava, SegWit transakcije se polako sve više i više koriste i  naknade za transakcije padaju, transakcije s $0.05 USD naknade su uključene odmah u sljedeći blok. Nema potrebe žuriti s hard forkom, koji ne treba shvaćati olako.
+
+Konsenzus zajednice je da se hard fork treba planirati godinu dana unaprijed, pa čak i dvije godine, ovisno o kompleksnosti. Pogotovo na sustavu čija vrijednost u vrijeme pisanja ovog članka iznosi ~100 milijardi USD.
+
+Uzmimo za primjer [Ethereum], nedavno je prošao kroz uspješan hard fork i implementaciju Byzantium nadogradnje o kojoj možete pročitati [ovdje][ethroadmap]. Taj hard fork je planiran od samog početka, tako da nije ni čudo da je sve prošlo bez problema.
+
+SegWit2x je zamišljen da se aktivira šest mjeseci nakon potpisivanja NYA dogovora, što je očito premalo vremena, pogotovo kad uzmemo u obzir da je tim koji radi na razvoju i testiranju gotovo ne postojeći.
+
+### Replay protection
+
+Vrlo bitna stavka kad se blockchain forka je zaštita ponavljanja transakcija na oba lanca.
+Da pojasnimo, blockchain do trenutka forka dijeli istu povijest (adrese, transakcije, [privatne][privatekey] i [javne][publickey] ključeve). 
+
+Nakon forka, privatni i javni ključevi su isti na oba forka, zato i dobijemo isti broj coina na drugom forku (Primjer: Bitcoin Cash). No, kad pošaljemo transakciju na lancu A, ona je potpisana našim privatnim ključem i kompletna je. Pošto transakcija ima sve potrebne elemente, netko može uzeti transakciju na lancu A i odaslati je na lancu B, tako da bi u slučaju da želimo prodati Bitcoin2x coine, šaljući njih na neku adresu, netko bi mogao uzeti našu Bitcoin2x transakciju i odaslati ju na originalnom Bitcoin lancu i tako bi mogli ostati bez Bitcoina.
+
+Zaštita protiv ponavljanja transakcija za SegWit2x ne postoji.
```

Comment:
Sada su na "opt in replay protection". Pojasniti.
Also dodaj link na Bitcoin Gold post u kojem je RP malo detaljnije pojašnjen.

Reply:
Koliko sam skuzio, imali su taj opt in RP, ali su ju onda maknuli.